pandas/core/arrays/base.py

```diff
@@ -611,9 +611,12 @@ def argmin(self):
         --------
         ExtensionArray.argmax
         """
+        validate_bool_kwarg(skipna, "skipna")
+        if not skipna and self.isna().any():
+            return -1
```

Comment:
those should not return -1, IMO. I would prefer to not put this "wrong" behaviour in the EAs (but keep it in Series were it already has this behaviour), as long as we didn't decide on the preferred behaviour.

Short term, could eg raise an error saying that skipna=False is not yet implemented, but by accepting `skipna`, it can already fix the default groupby case (see https://github.com/pandas-dev/pandas/pull/38733#issuecomment-756675205)

Reply:
> could eg raise an error saying that skipna=False is not yet implemented

so... like i had in the previous commit?

Reply:
> so... like i had in the previous commit?

Yes, that would be my preference.

(sorry for not being explicit about it, but my earlier comment of "this PR can already fix the non-skipna case in the meantime?" quite well matched the state of the PR, it was mainly to mean: we don't have to solve the skipna=False discussion to merge a PR like this to fix the skipna=True case)